src/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Levi's website | Home</title>
+    <title>Sleep outside | Home</title>
```

Comment:
I thought you were a native english speaker
fix it nerd

Reply:
I am the best English speaker you know